<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
          "DTD/xhtml1-strict.dtd">
<html>
  <head>
    <title>testtools.runtest : API documentation</title>
    <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
    <link href="apidocs.css" type="text/css" rel="stylesheet" />
    
    
  </head>
  <body>
    <h1 class="module">t.runtest : module documentation</h1>
    <p>
      <span id="part">Part of <a href="testtools.html">testtools</a></span>
      
      
    </p>
    <div>
      
    </div>
    <div class="docstring">
      <div>Individual test case execution.<table class="fieldTable"></table></div>
    </div>

    
    <div id="splitTables">
      <table class="children sortable" id="id132">
  
  
  <tr class="class">
    
    
    <td>Class</td>
    <td><a href="testtools.runtest.MultipleExceptions.html">MultipleExceptions</a></td>
    <td><span>Represents many exceptions raised from some operation.</span></td>
  </tr><tr class="class">
    
    
    <td>Class</td>
    <td><a href="testtools.runtest.RunTest.html">RunTest</a></td>
    <td><span>An object to run a test.</span></td>
  </tr><tr class="function private">
    
    
    <td>Function</td>
    <td><a href="testtools.runtest.html#_raise_force_fail_error">_raise_force_fail_error</a></td>
    <td><span class="undocumented">Undocumented</span></td>
  </tr>
</table>
      
    </div>
    
    

    <div class="function">
  <a name="testtools.runtest._raise_force_fail_error">
    
  </a>
  <a name="_raise_force_fail_error">
    
  </a>
  <div class="functionHeader">
    
    def
    _raise_force_fail_error():
    
  </div>
  <div class="docstring functionBody">
    
    <div class="undocumented">Undocumented</div>
  </div>
</div>
    <address>
      <a href="index.html">API Documentation</a> for <a href="https://github.com/testing-cabal/testtools">testtools</a>, generated by <a href="https://launchpad.net/pydoctor/">pydoctor</a> at 2015-05-10 23:16:58.
    </address>
  </body>
</html>